### ISSUE TITLE
file.io: Too many requests (429)

### ISSUE THREAD

**Issue by gohai** [googleworkspace/md2googleslides#155]
Running this command on the provided example file currently fails with an error from file.io:

`md2gslides example.md --title "Example" --use-fileio`

Details:

`Unable to generate slides: StatusCodeError: 429 - "{\"success\":false,\"status\":429,\"code\":\"TOO_MANY_REQUESTS\",\"message\":\"Too many requests\",\"help\":\"https://file.io/help/api/errors/TOO_MANY_REQUESTS\",\"key\":null}"`

This is with 928ef4ecb3f88c0d5ef3008a8e0f14e727ad54f8

**Comment by gohai** [comment]
(Looks like file.io currently only does 1 request/minute on their free plan.)

**Comment by wescpy** [comment]
Yep, either: 1) upgrade to a paid plan of theirs, 2) use an alternative service, or 3) live with 1rpm.

**Comment by jpoehnelt** [comment]
Closing this issue as the repository is archived.
